pom.xml

```diff
@@ -1231,6 +1231,7 @@
         <module>testsuite/testsuite-jolokia</module>
         <module>testsuite/testsuite-jolokia-keycloak</module>
         <module>testsuite/testsuite-keycloak</module>
+        <module>testsuite/testsuite-keycloak-mpjwt</module>
```

Comment:
I wonder whether `testsuite-keycloak` and `testsuite-keycloak-mpjwt` should be in the `core-testsuite` profile instead of `unsupported-testsuite`?

Reply:
I'll answer myself: they shouldn't, because they depend on `keycloak-server` :-)